app/views/hyrax/collections/show.html.erb

```diff
@@ -90,7 +90,7 @@
   <%= render 'shared/colored_divider', color: 'lime-green' do %>
     <a href="<%= view_all_url %>" class='btn btn-lg btn-primary collection-landing-button col-sm-10'>View All</a>
     <%# NEW BUTTON: Add in an edit button to collection rather than going through the dashboard %>
-    <% if user_signed_in? && current_user.admin? %>
+    <% if user_signed_in? && (current_user.admin? || current_ability.can?(:edit, @presenter.solr_document)) %>
```

Comment:
Patron users can also create collections & don't have access to the dashboard to edit their collections. You need to make sure this isn't a User Collection type collection. We usually do that with something like:
`collection_type.machine_id == 'user_collection'` You'll have to edit that to properly get the collection_type

Reply:
@CGillen How does the `user_collection` see on their end when they edit? Is it just this?

<img width="885" alt="Screenshot 2023-08-18 at 10 30 21 AM" src="https://github.com/OregonDigital/OD2/assets/111306182/236d8099-1b87-4412-9026-7fdd8afb7155">